convert2rhel/actions/report.py

```diff
@@ -26,6 +26,9 @@
 
 logger = logging.getLogger(__name__)
 
+#: The filename to store the results of running preassessment
+CONVERT2RHEL_JSON_RESULTS = "/etc/convert2rhel-assessment.json"
```

Comment:
I can see that for Leapp the txt report is being created at this path `/var/log/leapp/leapp-report.txt`. I suspect they generate the json to the same folder. They also generate the migration-results file to /etc/ but that serves quite a different purpose - it's in /etc/ to make it more likely to survive an attempt to purge the converted system from all-things-conversion. So I'd put our report under `/var/log/`.

Also, we use the word report instead of assessment everywhere, so I'd name it convert2rhel-report.json.

Reply:
Ok, let's wait for the answer from the Leapp team what are their plans around a post-upgrade report.

Because we want to add the post-conversion report (https://issues.redhat.com/browse/RHELC-902), it might be beneficial to differentiate the pre- and post- ones. Unless we use just one file for both reports.